google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/internal/TopicBacklogReader.java

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
This is probably not useful outside of dataflow, would you mind putting it in the beam directory?

Reply:
Moved, hopefully correctly